documentation/api/query/curl.markdown

```diff
@@ -81,3 +81,14 @@ If you do this with an endpoint that accepts `GET` requests, **you must also use
 
     curl -G http://localhost:8080/pdb/query/v4/nodes \
       --data-urlencode 'query=["=", ["node", "active"], true]'
+
+## Querying PuppetDB with POST
```

Comment:
Some of tutorial.md mentions GET explicitly in the "How to Query Section", maybe worth going over that, seeing if it needs a mention.